roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
The mojo doc and trello card seem to indicate that they will be consolidating, but origin will have different naming, so we'll still need to call a different service, it will just be if deployment_type == 'origin' or not.